src/OrleansCodeGenerator/CodeGeneratorCommon.cs

```diff
@@ -72,6 +72,18 @@ public static GeneratedAssembly CompileAssembly(GeneratedSyntax generatedSyntax,
             // Generate the code.
             var options = new CSharpCompilationOptions(OutputKind.DynamicallyLinkedLibrary);
 
+#if NETSTANDARD
```

Comment:
Yeah, it's fine either way really - not going to push back. I'm happy to merge as soon as build passes